ios-base/DroidKaigi 2020/Extentions/Observable+Ex.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import RxCocoa
+import RxSwift
+
+extension Observable where Element: OptionalType {
+    func filterNil() -> Observable<Element.Wrapped> {
```

Comment:
You can use `.compactMap`, if exclude nil value.
a.g. `collectionView.rx.itemSelected.compactMap { try dataSource.model(at: $0) as? Session }`

Reply:
I didn't notice it was made, thx!

Reply:
Removed in 1cf26c9e